config-overrides.js

```diff
@@ -48,13 +45,6 @@ module.exports = {
             }),
         );
 
-        // Add DefinePlugin to expose just the version
-        config.plugins.push(
-            new webpack.DefinePlugin({
-                'process.env.UI_VERSION': JSON.stringify(packageJson.version),
```

Comment:
what's wrong with this approach?

Reply:
`process.env` values are embedded in build, so in build I have
`version: process.env.UI_VERSION` -> `version: 8.11.0`

However, when we use it as a package, we have only `process.env.UI_VERSION` in our code, when we import it and try to open in UI we receive `process` is not defined error, no variables are exposed